### DISCUSSION TITLE
installing on xubuntu 18.04 x32

### DISCUSSION THREAD

**Discussion by goodcoderfromeurope** [openai/whisper#1401] [Q&A]
thomas@thomas-H55M-S2V:~$ pip install -U openai-whisper
Collecting openai-whisper

 Could not find a version that satisfies the requirement openai-whisper (from versions: )
No matching distribution found for openai-whisper

is it possible to run whisper on this setup?

**Comment by coastal45** [comment]
What is your Python version? If it's the same as Ubuntu 18.04, that version (3.6 if I recall) is not supported. Min. 3.8 according to Whisper read.me.

**Reply by goodcoderfromeurope** [reply]
I don't know. Gonna check, but still wanna use this as windows user. My linux skill isn't good.
